packages/demo/src/helpers/launchEditor.ts

```diff
@@ -76,6 +82,44 @@ export function launchEditor(currentSmartFilter: SmartFilter, engine: ThinEngine
                     currentSmartFilter.name + ".json"
                 );
             },
+            loadSmartFilter: async (file: File) => {
+                return smartFilterLoader.loadFromFile(file, false); // TODO: update optimize
+            },
+            customSave: async () => {
+                const serializer = new SmartFilterSerializer(
+                    blocksUsingDefaultSerialization,
+                    additionalBlockSerializers
+                );
+
+                const smartFilterJson = JSON.stringify(serializer.serialize(currentSmartFilter));
+
+                const dataToSend = {
+                    payload: JSON.stringify({
+                        smartFilter: smartFilterJson,
+                    }),
+                    name: "",
+                    description: "",
+                    tags: "",
+                };
+
+                const response = await fetch(smartFilterLoader.snippetUrl, {
```

Comment:
LOL. I misunderstood. I thought "no double hashes" == "ignore version, create a new snippet every time". My bad.
What should we use instead of the second hash to delimit the version in the url?